pkgs/applications/networking/browsers/chromium/browser.nix

```diff
@@ -12,11 +12,10 @@ mkChromiumDerivation (base: rec {
   sandboxExecutableName = "__chromium-suid-sandbox";
 
   installPhase = ''
-    mkdir -p "$libExecPath/swiftshader"
+    mkdir -p "$libExecPath/chromium"
     cp -v "$buildPath/"*.pak "$buildPath/"*.bin "$libExecPath/"
     cp -v "$buildPath/icudtl.dat" "$libExecPath/"
     cp -vLR "$buildPath/locales" "$buildPath/resources" "$libExecPath/"
-    cp -v "$buildPath/swiftshader/"*.so "$libExecPath/swiftshader/"
```

Comment:
Is this no longer needed? Does WebGL work for you?

Reply:
Ah, sorry... missed your comment about swiftshader being integrated, nevertheless, please try if WebGL works, because we had that broken a few weeks ago already.

Reply:
yeah, that `cp` broke the build (because the source folder was empty), so I removed it. in research, I found that thing about swiftshader being integrated and it figured. I'll try WebGL, as soon as the build finishes.